dev-tools/rpm/nfpm.yaml

```diff
@@ -10,9 +10,15 @@ maintainer: "devops@endgame.com"
 description: Elastic Fleet
 vendor: "Elastic NV"
 homepage: "http://www.elastic.co/"
-files:
-  ./bin/fleet: "/usr/bin/fleet"
-config_files:
-  ./systemd/fleet.service: "/usr/lib/systemd/system/fleet.service"
-  ./fleet-server.yml: "/usr/share/fleet/fleet-server.yml"
+contents:
```

Comment:
Not sure how these changes are related?

Reply:
This is not related, but the `make rpm` was broken, I just fixed it. Convenient when needed to build Linux binaries on Mac.